drivers/flash/flash_stm32_v1.c

```diff
@@ -0,0 +1,246 @@
+/*
+ * Copyright (c) 2017 BayLibre, SAS
+ * Copyright (c) 2019 Linaro Limited
+ * Copyright (c) 2020 Andreas Sandberg
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(flash_stm32generic, CONFIG_FLASH_LOG_LEVEL);
+
+#include <kernel.h>
+#include <device.h>
+#include <string.h>
+#include <drivers/flash.h>
+#include <init.h>
+#include <soc.h>
+
+#include "flash_stm32.h"
+
+#define WRITE_BLOCK_SIZE DT_PROP(DT_INST(0, soc_nv_flash), write_block_size)
```

Comment:
`WRITE_BLOCK_SIZE` identifier should be also used in initialization  of `flash_parameters.write_block_size`, it might not be good idea to get the same value with `DT_PROP` in multiple lines. This way someone may change the value of `WRITE_BLOCK_SIZE` here and miss the `flash_parameters` structure.

Reply:
Done